### PR TITLE
Handle exceptions by using traceback.format_exc instead of extract_tb

### LINKED ISSUE #57: Exceptions thrown in handler function make process hang
If the handler function throws an error, the execution of `lambda_call` never returns. A stack trace is printed containing the message `TypeError: Object of type FrameSummary is not JSON serializable`.

Looks like this is caused in line 127 of main.py (where you're setting `stackTrace`):
```python
        result = json.dumps({
            "errorMessage": str(err[1]),
            "stackTrace": traceback.extract_tb(err[2]),
            "errorType": err[0].__name__
        }, indent=4, separators=(',', ': '))
```
`json.dumps` is not happy with serializing an array of FrameSummary objects, which is what `extract_tb` returns.

### PR DESCRIPTION
**Pull request by danielthepope**
This fixes #57, where we were seeing `TypeError: Object of type FrameSummary is not JSON serializable` if the handler function throws an exception.

`traceback.format_exc` returns a string that can be easily turned into JSON.

### CODE REVIEW

**Comment by danielthepope** [comment]
I should also add a test to check that the command line execution completes with a non-zero exit code.

File: lambda_local/main.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ def execute(func, event, context):
         err = sys.exc_info()
         result = json.dumps({
             "errorMessage": str(err[1]),
-            "stackTrace": traceback.extract_tb(err[2]),
+            "stackTrace": traceback.format_exc(),
```
Comment (yxd-hde) [inline lambda_local/main.py]: How about using `traceback.format_tb(err[2])` instead?

Comment (danielthepope) [reply lambda_local/main.py]: Good idea, that makes it look more like the error response provided by AWS now.
